lib/bolt/transport/local/connection.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'open3'
+require 'fileutils'
+require 'bolt/node/output'
+require 'bolt/util'
+
+module Bolt
+  module Transport
+    class Local < Simple
+      class Connection
+        attr_accessor :user, :logger, :target
+
+        CHUNK_SIZE = 4096
+        PS_ARGS = %w[-NoProfile -NonInteractive -NoLogo -ExecutionPolicy Bypass].freeze
```

Comment:
These don't look to get used anywhere in this PR (sorry I'm going one at a time :grimacing:)